editor/styling/stylesheets/modules/_bundle-dialog.scss

```diff
@@ -31,7 +31,7 @@
     }
 
     .choice-box, .text-field {
-      -fx-min-width: 143px;
+      -fx-min-width: 165px;
```

Comment:
Hi again! Just tested this, and had to actually increase this to `180px` to get the field widths to align. This is quite strange, as I'm also running macOS and it looks good in the screenshots you posted. Did you perhaps forget to push a change?

Reply:
maybe it depends on display scale settings in mac os, I will check :)

Reply:
Also, since css doesn't get commited... Is it possible, that you got the old css right after checkout and forgot to recompile it from sass the first time, then did the change and then did the compilation?

Reply:
No, I'm pretty sure that can't be it. I ran `lein init`, and the Sass compiler runs as part of that. I also re-ran `lein sass` a couple of times and reloaded the stylesheet in order to reach the conclusion that `180px` was needed to make them align.

I don't think I've changed any DPI-related settings on my computer, but I'll double-check on another machine.

Reply:
Finally got it to build on my laptop as well. Looks great there! I think we could merge it as is as it is a minor alignment issue anyway - just wanted to make sure we weren't missing a commit.

My laptop runs Catalina, whereas my main iMac is still on Mojave. Maybe that could explain it?

Reply:
that's weird, you can merge this, but I'll still take some time to try and reproduce the issue a little bit later today, thanks!

Reply:
Thanks! I've merged it for now. Thank you again for implementing this feature! ❤️